### ISSUE TITLE
startPurchaseFlow method no resolved

### ISSUE THREAD

**Issue by leolim031** [googlecodelabs/play-billing-codelab#8]
As title  
![1](https://user-images.githubusercontent.com/31512132/37130632-629ce544-2252-11e8-83c3-6f2aa732570a.png)

**Comment by mickeymonroe** [comment]
Yes this is old but doing searches for this problem was limited and wanted to put the solution here. The problem is that when you are following the codelab instructions it is quite easy to copy and paste the code from step 4 into the BillingManager class and inadvertantly overwrite the startPurchaseFlow method which is located there. The startPurchaseFlow method is not in the  SkusAdapter class. Hope this helps someone....
Solution: add this method back in BillingManager class
 public void startPurchaseFlow(String skuId, String billingType) {
        // TODO: Implement launch billing flow here
    }
